agentai/api.py

```diff
@@ -37,16 +42,17 @@ def chat_complete(
         headers=headers,
         json=json_data,
     )
+    response.raise_for_status()
     if return_function_params:
         message = response.json()["choices"][0]
         if message["finish_reason"] == "function_call":
             return response
         else:
-            raise ValueError(f"Unexpected message: {message}")
+            raise Exception(f"Unexpected message: {message}")
```

Comment:
Too broad, add something specific e.g. ValueError or similar? Help the dev understand why this is wrong in someway

Reply:
Done. Using `VaueError`